### PR TITLE
feat: Add enabled or disabled of allow list in devops base info

### PR DESCRIPTION
**Pull request by harrisonliu5**
Signed-off-by: harrisonliu5 <harrisonliu@kubesphere.io>

### What type of PR is this?

/kind feature

### Which issue(s) this PR fixes:

Fixes #

### Special notes for reviewers:

https://user-images.githubusercontent.com/17872673/165935175-58500493-0151-4cfe-a830-bd647232bb94.mov

### Does this PR introduced a user-facing change?

```release-note
None
```

### Additional documentation, usage docs, etc.:

```docs

```

### CODE REVIEW

File: locales/en/l10n-devopsProjects-devopsProjectSettings-basinInformation.js [hunk 1]
```diff
@@ -36,4 +36,8 @@ module.exports = {
   MULTI_CLUSTER_PROJECT_NOT_FOR_CD: 'Multi-cluster projects cannot be used for continuous deployment.',
   CODE_REPOSITORY_EXIST_DESC: 'The code repository already exists. Please select another code repository.',
   DEPLOYMENT_LOCATION_EXIST_DESC: 'The deployment location already exists. Please set another deployment location.',
+  CODE_REPOSITORY_NOT_FOUND:'Code repository is not found',
+  RESOURCE_DEPLOYMENT_LOCATION_NOT_FOUND:'Resource deployment location is not found',
+  ALL_CODE_REPOSITORIES:'all code repositories',
```
Comment (LinuxSuRen) [inline locales/en/l10n-devopsProjects-devopsProjectSettings-basinInformation.js:41]: capitalize the first character?

**Review by LinuxSuRen — COMMENTED** [review]
I have tested it manually. It works well expect one small comment.

```
surenpi/ks-console:fix-argoCD@sha256:a0f37592f2a23d25ea0699f8b2fa923f115bcc4e75ac8e6c82f2c0b33452b64c
```

File: locales/en/l10n-devopsProjects-devopsProjectSettings-basinInformation.js (hunk 1 above)
Comment (harrisonliu5) [reply locales/en/l10n-devopsProjects-devopsProjectSettings-basinInformation.js:41]: /assign @Patrick-LuoYu

**Comment by harrisonliu5** [comment]
If the function meets the requirements, maybe could be merged first.

**Pull request merged**